src/main/java/seedu/sudohr/model/events/Event.java

```diff
@@ -0,0 +1,93 @@
+package seedu.sudohr.model.events;
```

Comment:
Thanks for really channeling all your energy into churning this out man - think you did plenty and overall quite nicely done too!